modules/ROOT/examples/live-demos/classic/index.js

```diff
@@ -48,7 +48,7 @@ const table2 = '<div> ' +
 
 const demoBaseConfig = {
   selector: 'textarea#classic',
-  width: 755,
+  width: "100%",
```

Comment:
I don't think this is a valid value. It's probably better to just leave this option out.

Reply:
String is an acceptable value according to this https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/api/OptionTypes.ts#L345

Reply:
I checked the code and it's only valid when it uses px